compiler/control/OptimizationPlan.hpp

```diff
@@ -39,7 +39,7 @@ namespace TR { class Recompilation; }
 namespace TR { class Monitor; }
 struct TR_MethodToBeCompiled;
 
-#if defined(__IBMCPP__)
+#if defined(__IBMCPP__) || defined(__open_xl__) && defined(__cplusplus)
```

Comment:
This file is C++, (e.g. see `namespace` above); please remove the redundant test of `defined(__cplusplus)`.